src/AppBundle/Membership/MembershipRequest.php

```diff
@@ -120,14 +120,24 @@ public function __construct()
         $this->conditions = false;
     }
 
-    public static function createWithCaptcha(string $recaptchaAnswer = null)
+    public static function createWithCaptcha(string $recaptchaAnswer = null): self
     {
         $dto = new self();
         $dto->recaptcha = $recaptchaAnswer;
 
         return $dto;
     }
 
+    public function setEmailAddress(string $emailAddress)
+    {
+        $this->emailAddress = mb_strtolower($emailAddress);
```

Comment:
@tgalopin I think so, it's more about the protocol, see the first result I got for searching "e-mail lower case" https://www.lifewire.com/are-email-addresses-case-sensitive-1171111.